include/cantera/thermo/ThermoPhase.h

```diff
@@ -1556,6 +1556,13 @@ class ThermoPhase : public Phase
             const Composition& oxComp, ThermoBasis basis=ThermoBasis::molar) const;
     //! @}
 
+    //! Return common parameters used by an equation of state that a user may
+    //! want to access.
+    virtual AnyMap getEosParameters()
```

Comment:
I'd like to suggest the name `getAuxiliary` or `getAuxiliaryData`, as I don't think this needs to be specific to retrieving equation of state parameters (and avoids having to stick the abbreviation "eos" into the method name).

I don't think I'd describe the parameters returned by this method as "common" -- at least for the current and any other use I can think of, it's more that they are intermediate, model-specific values in the equation of state calculation. Also, it would be helpful to mention where the user can find what parameters are retrieved for a given thermo model.

Reply:
That's a great name. I've updated to the getAuxiliaryData name, and changed the docstring.